api_app/script_analyzers/observable_analyzers/firehol_iplist.py

```diff
@@ -0,0 +1,91 @@
+import os
+import logging
+import traceback
+import requests
+import ipaddress
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import settings
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+db_path = f"{settings.MEDIA_ROOT}"
+
+
+class FireHol_IPList(classes.ObservableAnalyzer):
+    def set_config(self, additional_config_params):
+        self.list_names = additional_config_params.get("list_names", [])
+
+    def run(self):
+        ip = self.observable_name
+        result = {}
+
+        for list_name in self.list_names:
+            result[list_name] = False
+
+            self.check_iplist_status(list_name)
+
+            with open(f"{db_path}/{list_name}", "r") as f:
+                db = f.read()
+
+            db_list = db.split("\n")
+
+            for ip_or_subnet in db_list:
+                if ip_or_subnet != "":
+                    if ipaddress.ip_address(ip) in ipaddress.ip_network(ip_or_subnet):
```

Comment:
Since `ipaddress.ip_address(ip)` would be a constant value, it makes sense to initialize only once, outside the loop.

Reply:
Right! I overlooked it. Changing it. Thanks!